packages/mdc-top-app-bar/short/foundation.ts

```diff
@@ -46,20 +46,17 @@ export class MDCShortTopAppBarFoundation extends MDCTopAppBarBaseFoundation {
     }
 
     if (!this.adapter_.hasClass(cssClasses.SHORT_COLLAPSED_CLASS)) {
```

Comment:
Remove this check since it is already attached to target scroll event in component.